core/src/main/java/org/testcontainers/DockerClientFactory.java

```diff
@@ -39,6 +35,7 @@
 
     private static final List<DockerConfigurationStrategy> CONFIGURATION_STRATEGIES =
```

Comment:
@rnorth - don't - beta9 literially just got released and they're killing `docker.local` and making it `localhost` ;-)

<img width="624" alt="docker_local" src="https://cloud.githubusercontent.com/assets/1248633/14900656/9a8478ee-0d89-11e6-9e0a-48939f8a166a.png">